copying.txt

```diff
@@ -1919,7 +1919,7 @@ docs/license.html_lib.
 This applies to Python Windows extensions, available from http://sourceforge.net/projects/pywin32/
 
 ```
-Unless stated in the specfic source file, this work is
+Unless stated in the specific source file, this work is

```

Comment:
it appears we no longer use or depend on this library (pywin32).
The library also has changed licence to the [Python Software Foundation licence](https://pypi.org/project/pywin32/) which is also included in this file.
I suggest you remove the entire "Python Windows Extensions License" section from this file.